### ISSUE TITLE
Theme colors

### ISSUE THREAD

**Issue by mertuarez** [ClosedXML/ClosedXML#98]
Can you implement real theme colors?

dSpreadsheet.WorkbookPart.ThemePart.Theme.ThemeElements.ColorScheme.Accent1Color

https://github.com/ClosedXML/ClosedXML/blob/daa413d77cf94b2fc026b68938d7e7b1c301753c/ClosedXML/Excel/XLWorkbook.cs#L330

**Comment by igitur** [comment]
Have a look at https://github.com/ClosedXML/ClosedXML/wiki/Using-Colors and see if it fits your needs.

**Comment by mertuarez** [comment]
I can change document theme in office [http://thewindowsclub.thewindowsclubco.netdna-cdn.com/wp-content/uploads/2013/04/Pick-Color.png]

I Mean:
Excel2013 Accent4 = FFC000 or custom
ClosedXML Accent4 = 8064A2 

ClosedXML has hardcoded scheme (similiar to office 2007-2010).

**Comment by jahav** [comment]
I am closing this, because theme colors have been implemented in #2117 / 0.103.
E.g. this code changes color of scheme, uses it in a cell and when workbook is opened, the color of the text changes along with selected theme (Page Layout - Themes - Colors dropdown).


```csharp
using var wb = new XLWorkbook();
var ws = wb.AddWorksheet();
wb.Theme.Text2 = XLColor.Red;

ws.Cell("F1")
	.SetValue("Hello World")
	.Style.Font.SetFontColor(XLColor.FromTheme(XLThemeColor.Text2));

wb.SaveAs(@"c:\Temp\Issues\0098\a.xlsx");
```
